sickbeard/tv.py

```diff
@@ -1100,9 +1100,16 @@ def refreshDir(self):
                     with curEp.lock:
                         # if it used to have a file associated with it and it doesn't anymore then set it to sickbeard.EP_DEFAULT_DELETED_STATUS
                         if curEp.location and curEp.status in Quality.DOWNLOADED:
+                        
+                            if sickbeard.EP_DEFAULT_DELETED_STATUS == 'ARCHIVED':
```

Comment:
config is a integer:. In my case is 6

so here should be == ARQUIVED without quotes, isn't?